<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamsonNodeWidget</class>
 <widget class="QWidget" name="SamsonNodeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Worker 1</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Memory</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="memory_description">
          <property name="text">
           <string>[]</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QProgressBar" name="memory_progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Process</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="process_description">
          <property name="text">
           <string>[]</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QProgressBar" name="process_progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Disk</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="disk_description">
          <property name="text">
           <string>[]</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QProgressBar" name="disk_progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Network</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="network_description">
          <property name="text">
           <string>[]</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QProgressBar" name="network_progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
